src/components/Postiterie.vue

```diff
@@ -146,16 +152,28 @@ export default {
     },
 
 
-
     // function resultToXML
     // TODO
     // return the XML from the result array[];
     resultToXML: function(){
-      let xml = "";
-    
+      let $this = this;
+      let xml = process.env.VUE_APP_XML_ENTETE;
+      //let xml = '<?xml version="1.0" encoding="UTF-8"?><mxGraphModel dx="1190" dy="757" grid="1" gridSize="10" guides="1" tooltips="1" connect="1" arrows="1" fold="1" page="1" pageScale="1" pageWidth="827" pageHeight="1169" math="0" shadow="0"><root><mxCell id="0" /><mxCell id="1" parent="0" />';
+      this.temps.forEach(function (el, index) {
+        xml+=$this.forXml(el, index)
+      })
+      xml+='</root></mxGraphModel>'
       return xml;
     },
 
+    forXml(el, index) {
```

Comment:
Probably poorly named. No idea what it does.

Reply:
Changed